modules/kargoBidAdapter.js

```diff
@@ -106,6 +110,12 @@ export const spec = {
       bidResponses.push(bidResponse);
     }
 
+    // If we got a bid and it's enabled, send response time data
+    if (bidResponses.length > 0 && bids[Object.keys(bids)[0]].logResponseTime) {
+      const responseTime = Date.now() - bidRequest.start;
+      this._sendResponseTimeData(bidRequest.auctionId, bidRequest.timeout, responseTime, 0);
```

Comment:
I do not think the prebid rules committee wants bid adapters to trigger pixels in the `onBidResponse` or `buildRequest` functions. I will check and get back to you.

Reply:
Thanks for the update, @robertrmartinez . I've gone ahead and removed this block from the update.